### DISCUSSION TITLE
Overlay for OBS Settings?

### DISCUSSION THREAD

**Discussion by AlieDupuis** [OhMyGuus/BetterCrewLink#23] [Q&A] [answered]
Hey There! Looking to use BetterCrewLink for use in my streaming, and I love the option to have a browser source for the CrewLink Overlay.

When I go to set it up, the default size for the overlay is 800 x 600. Should I set this to be the default pixel size of my screen for it to accurately capture the overlay? Is there a default size I should make it?

Thanks so much  for making this BetterCrewLink fork :)

**Comment by OhMyGuus** [comment, marked as answer]
This needs to be the in-game game resolution.
You can follow this image to set it up correctly.

https://i.imgur.com/zjvevBu.png

If it still doesn't work u can join the discord or msg me ThaGuus#2140
